tests/unittest/_torch/thop/serial/test_moe.py

```
@@ -38,6 +38,25 @@
     reorder_rows_for_gated_act_gemm, shuffle_matrix_a, shuffle_matrix_sf_a)
 
 
+@contextmanager
```

Ruff (0.16.2)

[error] Undefined name `contextmanager`

(F821)